.github/workflows/test.yml

```diff
@@ -23,7 +23,6 @@ jobs:
     strategy:
       matrix:
         include:
-          - python: "3.7"
```

Comment:
@zerolab would you document this change in the changelog, please?

Reply:
it is already in 1.5 - https://github.com/wagtail/Willow/blob/main/docs/changelog.rst?plain=1#L7

Reply:
@zerolab that doesn't state Python 3.7 support was dropped. It states support for all versions _before_ 3.7 is dropped.

Reply:
The wording in that changelog entry is a bit ambitious, easily misinterpreted

Reply:
🤦🏼 yup. Can you tell I am too eager?

Reply:
That's why having a second pair of eyes is helpful 🙂

Reply:
5de38a6